### ISSUE TITLE
Weird debug issues -- out of the box

### ISSUE THREAD

**Issue by tmpg** [jedwards1211/meteor-webpack-react#142]
I do not know if this is directly related to this app or if it is a general error but it tried on 2 different systems and got the same issue:

> Your Node version (v0.10.43) has a partial support of profiler.
> The stack frames tree doesn't show all stack frames due to low sampling rate.
> The profiling data is incomplete and may show misleading results.
> Update Node to v0.11.13 or newer to get full support.
> 

On both I have node v6.0.0 installed and tried with many others ranging from .12 through 6. Is the debug feature worth stressing over?

Thanks!

**Comment by jedwards1211** [comment]
Yes, I've seen that.  The problem is Meteor picks its own version of Node to run, regardless of what you have installed.  Hopefully they'll stop doing that soon after the Meteor 1.3 transition to being npm packages.  Other than this warning, the debugger seems to work fine.  I can't remember exactly why I made `debug.js` work differently than typical `meteor debug`, but I know there was some reason...maybe I needed `--debug-brk`?
